RadixWallet/Features/SettingsFeature/ManualAccountRecoveryScan/ManualAccountRecoverySeedPhrase+Reducer.swift

```diff
@@ -0,0 +1,169 @@
+import ComposableArchitecture
+import SwiftUI
+
+// MARK: - ManualAccountRecoverySeedPhrase
+public struct ManualAccountRecoverySeedPhrase: Sendable, FeatureReducer {
+	public typealias Store = StoreOf<Self>
+
+	// MARK: - State
+
+	public struct State: Sendable, Hashable {
+		@PresentationState
+		public var destination: Destination.State? = nil
+
+		public var isOlympia: Bool
```

Comment:
Meant to be mutable?

Reply:
This should not be here at all, for a second I was thinking of storing the olympia/babylon state here and update when they press a button for either of those, but I couldn't bring myself to do such a thing. Just forgot to remove it.

Reply:
probably not, @kugel3 Im changing it to a `let`, we never wrote to it....

Reply:
fixed in https://github.com/radixdlt/babylon-wallet-ios/pull/941